app/src/main/java/com/sun/basic_japanese/data/model/Alphabet.kt

```diff
@@ -13,12 +12,22 @@ data class Alphabet(
     var remember: Int
 ) {
     constructor(cursor: Cursor) : this(
-        cursor.getInt(cursor.getColumnIndex(AppDatabase.DATABASE_COLUMN_ID)),
-        cursor.getString(cursor.getColumnIndex(AppDatabase.DATABASE_COLUMN_ROMAJI)),
-        cursor.getString(cursor.getColumnIndex(AppDatabase.DATABASE_COLUMN_HIRAGANA)),
-        cursor.getString(cursor.getColumnIndex(AppDatabase.DATABASE_COLUMN_KATAKANA)),
-        cursor.getString(cursor.getColumnIndex(AppDatabase.DATABASE_COLUMN_AUDIO)),
-        cursor.getInt(cursor.getColumnIndex(AppDatabase.DATABASE_COLUMN_GROUPE)),
-        cursor.getInt(cursor.getColumnIndex(AppDatabase.DATABASE_COLUMN_REMEMBER))
+        cursor.getInt(cursor.getColumnIndex(DATABASE_TABLE_ALPHABET_COLUMN_ID)),
+        cursor.getString(cursor.getColumnIndex(DATABASE_TABLE_ALPHABET_COLUMN_ROMAJI)),
+        cursor.getString(cursor.getColumnIndex(DATABASE_TABLE_ALPHABET_COLUMN_HIRAGANA)),
+        cursor.getString(cursor.getColumnIndex(DATABASE_TABLE_ALPHABET_COLUMN_KATAKANA)),
+        cursor.getString(cursor.getColumnIndex(DATABASE_TABLE_ALPHABET_COLUMN_AUDIO)),
+        cursor.getInt(cursor.getColumnIndex(DATABASE_TABLE_ALPHABET_COLUMN_GROUPE)),
+        cursor.getInt(cursor.getColumnIndex(DATABASE_TABLE_ALPHABET_COLUMN_REMEMBER))
     )
+
+    companion object {
+        const val DATABASE_TABLE_ALPHABET_COLUMN_ID = "id"
```

Comment:
Ngắn gọn như vậy là đủ nghĩa.
```suggestion
        const val COL_ID = "id"
```